{
  "imports": {
    "class": "imports",
    "CISCO-SMI": [
      "ciscoAgentCapability"
    ],
    "SNMPv2-CONF": [
      "AGENT-CAPABILITIES",
      "MODULE-COMPLIANCE",
      "NOTIFICATION-GROUP"
    ],
    "SNMPv2-SMI": [
      "IpAddress",
      "Integer32",
      "TimeTicks",
      "Unsigned32",
      "OBJECT-IDENTITY",
      "iso",
      "MibIdentifier",
      "Gauge32",
      "Bits",
      "Counter32",
      "MODULE-IDENTITY",
      "Counter64",
      "NOTIFICATION-TYPE",
      "OBJECT-TYPE"
    ],
    "SNMPv2-TC": [
      "TEXTUAL-CONVENTION",
      "DisplayString"
    ]
  },
  "ciscoLinkErrorMonitorCapability": {
    "name": "ciscoLinkErrorMonitorCapability",
    "oid": "1.3.6.1.4.1.9.7.414",
    "class": "moduleidentity",
    "revisions": [
      {
        "revision": "2004-08-05 00:00",
        "description": "Initial version of this MIB module."
      }
    ]
  },
  "clemCapCatOSV08R0401": {
    "name": "clemCapCatOSV08R0401",
    "oid": "1.3.6.1.4.1.9.7.414.1",
    "class": "agentcapabilities",
    "productrelease": "Cisco CatOS 8.4(1).",
    "status": "current"
  },
  "meta": {
    "comments": [
      "ASN.1 source file:///home/runner/work/mibs/mibs/src/vendor/cisco/CISCO-LINK-ERROR-MONITOR-CAPABILITY",
      "Produced by pysmi-1.1.8 at Sun Jan 16 15:51:05 2022",
      "On host fv-az39-968 platform Linux version 5.11.0-1025-azure by user runner",
      "Using Python version 3.10.1 (main, Dec 22 2021, 10:45:09) [GCC 9.3.0]"
    ],
    "module": "CISCO-LINK-ERROR-MONITOR-CAPABILITY"
  }
}